portfolio/src/main/webapp/script.js

```diff
@@ -12,25 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-
-let index = 0;
+/*
+    The function showNext() allows the button to be clicked twice in order to
+    display a personal summary and a photograph. Because the second click is dependent
+    on the first, a global variable is required to determine when to show the picture.
+    Another roundabout solution would be to check whether the 'summaryCointainer'
+    is empty, eliminating the need for a global varible.
+*/
+var summaryVisble = false;
+var pictureVisbile =false;
 function showNext() {
-    index = index +1;
-    if(index ==2){
-        const imgUrl = 'Me.jpg';
-        const imgElement = document.createElement('img');
-        imgElement.width = '550';
-        imgElement.height= '450';
-        imgElement.src = imgUrl;
-        const imageContainer = document.getElementById('picture');
-        imageContainer.appendChild(imgElement);
-
-    }
-    if(index == 1){
-        const summaryContainer = document.getElementById('summary');
-        summaryContainer.innerText="Currently, I'm a 2nd year student at the Rose-Hulman Institute of Technology working towards a bachelor in Computer Science. I grew up in Louisville, Kentucky and moved on to college early. I completed two years at Western Kentucky University while simultaneously obtaining my highschool diploma. The summer of my graduation I was accepted as a Google CSSI student and spent 3 weeks at the Google Cambridge Office. A year later and I am now working as a virtual Google intern in my hometown of Louisville."
+    if(summaryVisble == false){
+        showSummary();
+        summaryVisble = true;
+    } 
+    else if(pictureVisbile == false){ 
+        showPicture();
+        pictureVisbile = true;
```

Comment:
More formatting that clang-format could (hopefully) catch: spacing after `if` and before `{`, indent 2 spaces, no line break between `}` and `else`.

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace
https://google.github.io/styleguide/jsguide.html#formatting-block-indentation
https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks